app/services/proofing/mock/ddp_mock_client.rb

```diff
@@ -1,27 +1,38 @@
 module Proofing
   module Mock
-    class DdpMockClient < Proofing::Base
-      vendor_name 'DdpMock'
+    class DdpMockClient
+      class << self
+        def vendor_name
+          'DdpMock'
+        end
 
-      required_attributes :threatmetrix_session_id,
-                          :state_id_number,
-                          :first_name,
-                          :last_name,
-                          :dob,
-                          :ssn,
-                          :address1,
-                          :city,
-                          :state,
-                          :zipcode,
-                          :request_ip
+        def required_attributes
```

Comment:
[suggestion] you might consider the more readable `%i` notation to generate an array of symbols here